### ISSUE TITLE
Accessibility feature request: Notification reminder

### ISSUE THREAD

**Issue by username-computer** [ubports/ubuntu-touch#1725]
Hello mighty fine people,  

My old folk wanted a phone, and i wanted them to be safe from adversary actors.  
So an Ubuntu touch phone is excelent and they love the simplicity of the user interface.  
However they have hearing problems and doesn't have the mindset to check the phone every so often.  
They don't notitice the notifications when one arrives.  
So a reminder that a notification has not yet been seen would be a god-send feature.  

### Description of the feature
A timely notification-sound that reminds the user of notifications where the user has not yet acted upon.  
  
This should be catagorized as an accessibility function.

#### Accessiblity button mockup  
This is based on the Universal Access button in the settings of Ubuntu.
![Accessibility_button](https://user-images.githubusercontent.com/83532512/116807844-cfa14080-ab35-11eb-9308-28e2fbb0fd95.png)

#### Notification reminder user interface
Blatantly based of the System Settings > [Personal] > Notifications menu  
What i love about this menu is that the user can manage what Apps can notify.  
This can be used as a base for the Notification Reminder function.  
The neccessary feature is the time setting so the user can set the notification reminder interval.
![notification_reminder_mockup](https://user-images.githubusercontent.com/83532512/116807905-05462980-ab36-11eb-98ce-a005cc94b8f5.png)

Thanks for reading.  
Keep up the good work.

**Comment by username-computer** [comment]
#### My stab at the feature
I made a [python script](https://github.com/ubports/ubuntu-touch/files/6411107/notify.py.txt "notify.py") that sends notifications to the dbus with [dbus-python](https://dbus.freedesktop.org/doc/dbus-python/ "https://dbus.freedesktop.org/doc/dbus-python/")
When i executed the script the screen of the phone lighted up and an generic notification was seen.  
Sadly the notification did not let the phone ring the notification sound.  

Also the notification didn't appear in the notification list of the quick access pull-down menu.  
This is a good thing but was not as expected.
This differs from what i have seen on Ubuntu desktop (20.04 LTS).  
The notification was seen and arrived in the notification list seen with key-combo: Super + M
  
#### Screenshot of the notification on the phone
![generic_notification](https://user-images.githubusercontent.com/83532512/116808825-278e7600-ab3b-11eb-8e34-407992ea0d3b.png)

**Comment by cibersheep** [comment]
Thanks for the suggestion.
Notifications will stay until dismissed in the notifications indicator, and a led will blink (check which devices have working LED here: https://devices.ubuntu-touch.io/ )

Your suggestion might be better as an app. You already have the python script done, you just need to add a UI and publish it in the store. Read more: https://clickable-ut.dev/

**Comment by username-computer** [comment]
@cibersheep 
> Notifications will stay until dismissed in the notifications indicator, and a led will blink (check which devices have working LED here: https://devices.ubuntu-touch.io/ )

The persistent notifications are not the point.
The reminder of those notifications are the main point for the feature request.

Long feature-request short:
As a user i want to be reminded of my available notifications by replaying the notification-sound each X amount of time.


@cibersheep 
> Your suggestion might be better as an app. You already have the python script done, you just need to add a UI and publish it in the store. Read more: https://clickable-ut.dev/

Sure cool, but there are 3 problems i have pointed out:
1. The dbus made notification doesn't replay the notification-sound.
2. The notification list that shows the available notifications can't be accessed/requested by the dbus
3. There is not a way to trigger the notification sound by sending a request onto the dbus

If i'm wrong, please collaborate how i'm wrong.

Personally i think this feature could be a unique selling point.
I have yet to see a mobile device with this feature.
And the ease how older people use Ubuntu-Touch is amazing.

**Comment by cibersheep** [comment]
Sorry, I didn't understand it at first. You are looking for a sound notification. Mmm.

I still believe is a very specific use case indeed. Better as an app than baked in the OS.
